### PR TITLE
Flatpak: Dependency update

### PR DESCRIPTION
**Pull request by TheComputerGuy96**

### CODE REVIEW

**Comment by deepbluev7** [comment]
@labhub new-pipeline

File: im.nheko.Nheko.yaml [hunk 1]
```diff
@@ -100,86 +100,90 @@ modules:
       - -Dvapi=false
       - -Dgtk_doc=false
       - -Dintrospection=false
-      # https://gitlab.gnome.org/GNOME/libsecret/-/issues/49
-      - -Dgcrypt=false
     sources:
       - commit: 3fe635e64efd4b8dbc9ec3548b0bc8034c7665c4
```
Comment (deepbluev7) [inline im.nheko.Nheko.yaml:104]: You also need to update the commit, otherwise this will fail to download, see the CI failures.

Comment (TheComputerGuy96) [reply im.nheko.Nheko.yaml]: I think I forgot to update it before (anyway this should be fixed now)

**Comment by deepbluev7** [comment]
This still seems to have failed the heif build, could you rebase it and check, that it actually builds?

**Comment by TheComputerGuy96** [comment]
@deepbluev7 Flatpak builds should now succeed (the pipeline failure is because of macOS and Windows jobs which are broken for unrelated reasons) :frog:

**Comment by deepbluev7** [comment]
Sorry, but I don't know what that command means.

Thanks for asking 🥰

**Comment by deepbluev7** [comment]
Thank you!

**Pull request merged**